mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -2862,8 +2876,80 @@ struct LLVMInlinerInterface : public DialectInlinerInterface {
             return false;
           return true;
         })
+        .Case<LLVM::CallOp>([](auto) { return true; })
         .Default([](auto) { return false; });
   }
+
+  // Copied almost verbatim from `FuncInlinerInterface::handleTerminator`.
```

Comment:
NIT: `///` instead of `//`. Same further down

Reply:
This is not meant as documentation though, it's meant as an inline comment 🤔

Reply:
Fair point. @gysit do you have some thought about this?

Reply:
I can also just get rid of this? At this point we also changed it a bit (by improving the coding style).

Reply:
Maybe have a quick doc comment what the function does and then hide below it is copy pasta :)